### DISCUSSION TITLE
Lamellensteuerung

### DISCUSSION THREAD

**Discussion by d8421** [brutella/hkknx-public#290] [Q&A]
Moin,

gibt es derzeit eine elegante Lösung die Lamellensteuerung im Rollladen-Modul nicht in den Geräteeinstellungen zu "verstecken"? Könnte man zB eine gesonderte Kachel mit Prozentschieber einführen, ähnlich wie der Ein/Aus-Schalter für die Sperrfunktion? Für Raffstoren ist die Lamellensteuerung sehr relevant.

Danke und viele Grüße
David

**Comment by brutella** [comment]
Wie Rollläden mit Lamellensteuerung angezeigt werden, hängt von der App ab. Die Apple Home-App zeigt die Lamellensteuerung in den Geräteeinstellungen an – daran kann hkknx nichts ändern. 

Welchen Datentyp verwendest du für die Lammellensteuerung und wie werden die Werte gemappt?
- 0% → -90°?
- 50% → 0°?
- 100% → +90°?

**Reply by d8421** [reply]
Danke, das ist mir klar. Meine Frage war eher, ob man ggf. ein anderes Format verwendet, um die Lamellensteuerung auf eine separate Kachel zu bringen, z.B. einen Schalter oder so etwas. Also kein Request im engeren Sinne, sondern eher die Frage nach einer cleveren Anwendungspraxis. 

Datentyp für Lamellen sind Prozentangaben. Das Mapping erfolgt innerhalb des Aktors über die Zeitmessung. Es ist die Zeit von der voll verschlossenen (nach unten geklappten Lamelle (vermutlich 175°) =100% zur maximal nach oben geklappten Lamelle (physisch vermutlich ca. 10°) = 0%.

Deshalb sind die Gradzahlen letztlich irrelevant; sie hängen ja auch maßgeblich von der Bauform der Raffstore ab. Maßgeblich ist allein die Verfahrzeit für eine solche "Wendung"; sie beträgt in meinem Fall zB 1,4 Sekunden. Der Aktor stellt dann die Lamelle durch sehr kruze Schritte her.
